apps/site/pages/docs/getting-started/settingup-project.mdx

```diff
@@ -47,40 +44,226 @@ tools in your machine:
   </p>
 </details>
 
-<details>
-  <summary>
-    <b>Code Editor</b>
-  </summary>
-  <p>
-    TBD
-  </p>
-</details>
+<Callout type="info" emoji="ℹ️">
+  Although you might opt to use an alternative code editor, note that Visual Studio Code might help you follow along with the FastStore documentation since our documentation may include screenshots from VS Code.
+</Callout>
 
-## Step 2: Clonning the Store's Repository
-1. Clone your store's repository that was created during the [Onboarding](/tbd) by running the following comand:
+## Step 1: Cloning the Store's Repository
+1. Clone your store's repository that was created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
 ```bash
     git clone https://github.com/vtex-sites/[store-name]
-```
\ No newline at end of file
+```
+<Callout type="warning" emoji="⚠️">
+Replace `[store-name]` for your store's repository.
+</Callout>
+
+2. Change into the working directory (e.g., cd `faststore.store`) and install all the dependencies listed within the `package.json` file:
+
+```bash
+cd faststore.store
+yarn install
+```
+
+3.  Open up your FastStore project in any code editor of your choice.
+
+<Callout type="info" emoji="ℹ️">
+  Note that your store configuration was already set during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created). Therefore, there is no need to connect your project to your VTEX account. 
+  If you want to review these configurations, you can access the `faststore.config.js` file in the root directory of your project's repository.
+</Callout>
+
+## Step 2: Running a local server
+
+Let's check what the project looks like in a web browser so far.
+
+1. Start a local development server to serve your website:
+
+```bash
+  yarn dev
+```
+This may take a few minutes. Once your development server is ready, the command line will output a similar message as the following:
+
+```bash
+  ready - started server on 0.0.0.0:3000, url: http://localhost:3000
+  event - compiled client and server successfully in 333 ms (951 modules)
+```
+<Callout type="info" emoji="ℹ️">
+  The `dev` command is a part of the FastStore CLI, a pre-configured command line interface in your starter project. This command enables you to execute code to keep your store's project up-to-date with the [@faststore/core package](/docs/getting-started/project-structure) using the CLI. For more information, you can refer to the topic on [FastStore CLI](/docs/getting-started/faststore-cli).
+</Callout>
+
+2. Now, open your favorite web browser and navigate to the localhost served, for example, http://localhost:3000.
+
+![localhost-example](https://vtexhelp.vtexassets.com/assets/docs/src/starter-gif___42003e72ece61064991a34c3f14e398f.gif)
+
+*Congratulations!* You have just created your very first FastStore store website! 🎉
```

Comment:
I might have missed it, but is it a good idea to tell people where this content is coming from?

Reply:
@gvc , do you mean saying that this content comes from the `@faststore/core` package, for example?